react/react_forms/README.md

```diff
@@ -1,387 +1,477 @@
   }
-}
+};
 ```
 
-> You can see the final product in the `solution` branch on the example repo.
-
-## Uncontrolled components
+## Uncontrolled Components
 
-So that's a fair amount of work, so let's look at another way to do this - using the uncontrolled component method.
+Controlled components seem like a fair amount of work, so let's look at another way to do this - using the uncontrolled components.
 
-The main difference here is that we won't be capturing the state of the input fields while they're changing. Only when the form gets submitted do we extract the values.
+The main difference here is that we won't be capturing the state of the input fields in our React component while they're changing. We rely on the document to keep track of what the user is doing - React doesn't have to be involved. We only extract and use the values in React when the form is actually submitted.
 
 Remove all the `onChange` and `values` attributes from all the inputs in the form. The only thing that should remain is the `onSubmit` function in the `<form>` tag itself.
 
-The component should render something like this:
+The component should look something like this:
 
 ```jsx
-return (
-  <form onSubmit={handleFormSubmit}>
-    <h2>User Information</h2>
-    
-    <input id="not-robot" type="checkbox" name="notARobot"/>
-    <label htmlFor="not-robot">I am not a robot</label>
-
-    <select name="title">
-      <option value=''></option>
-      <option value='mr'>Mr.</option>
-      <option value='ms'>Ms.</option>
-      <option value='mrs'>Mrs.</option>
-      <option value='mx'>Mx.</option>
-      <option value='dr'>Dr.</option>
-    </select>
-
-    <input
-      type="text"
-      placeholder="First Name"
-    />
-    <input
-      type="text"
-      placeholder="Last Name"
-    />
-
-    <button type="submit">Submit</button>
-  </form>
-)
+render() {
+  return (
+    <form onSubmit={this.handleFormSubmit} className="form-container">
+      <h2>User Information</h2>
+      <input
+        id="not-robot"
+        type="checkbox"
+      />
+      <select>
+        <option value=""></option>
+        <option value="mr">Mr.</option>
+        <option value="ms">Ms.</option>
+        <option value="mrs">Mrs.</option>
+        <option value="mx">Mx.</option>
+        <option value="dr">Dr.</option>
+      </select>
+      <input
+        type="text"
+        title="firstName"
+        placeholder="First Name"
+      />
+      <input
+        type="text"
+        title="lastName"
+        placeholder="Last Name"
+      />
+      <button type="submit">Submit</button>
+    </form>
+  );
+}
 ```
 
-Now we have to give every input a `name` attribute. The name will be how we access the form values, so they need to be unique but only within this form.
+Now, just like when we had to apply a `title` to our `text` inputs, we have to give every input a `name` attribute. The name will be how we access the form values, so they will each need to be unique:
```

Comment:
Why can't we use title again? Or why couldn't we use name before?

Reply:
Because we're cutting this section, it doesn't matter, but I'll change `title` to `name` in the other section - good catch.